microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -21,40 +21,86 @@ In your performance recordings, some of these long-running tasks may be **Recalc
 
 When you have long-running **Recalculate Style** events, you can use the **Selector Stats** feature to understand which of your CSS selectors are taking up the most time and resulting in slow performance.
 
-## How to use Selector Stats
+The **Selector Stats** feature provides statistics about the CSS rule selectors that were involved in one or more **Recalculate Style** events within a performance recording.
 
-To see **Selector Stats** during **Recalculate Style** events:
+## View CSS rule selector statistics
+
+#### Record a performance trace with Selector Stats enabled
+
+To view the statistics of your CSS rule selectors during long-running **Recalculate Style** events, first record a performance trace with the Selector Stats feature enabled.
+
+The Selector Stats feature isn't always enabled because it adds more overhead to your performance recordings. You should only leave it turned on when you need to investigate the performance of **Recalculate Style** events  and other rendering information:
+
+To record a performance trace with selector statistics:
 
 1. In the **Performance** tool, click **Capture settings** (![Capture settings icon](./selector-stats-images/capture-settings-icon.png)).
 
-1. Click **Enable advanced rendering instrumentation (slow)** to turn the setting on. Note that this setting adds more overhead for performance recordings, so you should only leave it turned on when you need to record **Selector Stats** and other rendering information.
+1. Click **Enable advanced rendering instrumentation (slow)** to turn the setting on:
+
+   ![The Enable advanced rendering instrumentation checkbox in the Performance tool](./selector-stats-images/enable-feature.png)
```

Comment:
```suggestion
   ![The 'Enable advanced rendering instrumentation' checkbox in the Performance tool](./selector-stats-images/enable-feature.png)
```
d/k about screen readers, but I previously tested this use of single-quotes by using Inspect.